http/src/main/resources/init-resources/azure_vm_init_script.sh

```diff
@@ -42,13 +42,17 @@ CONTENTSECURITYPOLICY_FILE=$8
 
 # Envs for welder
 WELDER_WSM_URL=${9:-localhost}
-WELDER_WORKSPACE_ID="${10:-dummy}"
-WELDER_STORAGE_CONTAINER_RESOURCE_ID="${11:-dummy}"
+WORKSPACE_ID="${10:-dummy}"
```

Comment:
Why the name change here? If these variables are shared across services, not just Welder maybe we should rename everything? Or leave a comment explaining what this corresponds to?

Reply:
Rob suggested a name change over me declaring another variable with the same value.
I'm not sure if the other variables will be used for anything else, or are applicable. I could guess the email, bucket, and storage container resource id may be generic? 🤷

Reply:
I'll add a comment, but I'm not sure how descriptive I can really be